### PR TITLE
TTB integrated x-sec datasets

### PR DESCRIPTION
**Pull request by t7phy**

### CODE REVIEW

File: buildmaster/CMS_TTBAR_8TEV_TOT/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        zero: {description: "", label: '$$', units: ""}
+        mt_sqr: {description: "mass of top quark squared", label: '$m_{t}^2$', units: '$GeV^2$'}
+        sqrt_s: {description: "center of mass energy", label: '$\sqrt{s}$', units: "$GeV$"}
+      file: kinematics.yaml
+    data_central: data.yaml
+    data_uncertainties:
+      - uncertainties.yaml
+    kinematic_coverage: [zero, mt_sqr, sqrt_s]
+    plotting:
+      dataset_label: '$Total cross section of t\bar{t} at 8 TeV$'
```
Comment (scarlehoff) [inline buildmaster/CMS_TTBAR_8TEV_TOT/metadata.yaml:36]:
```suggestion
      dataset_label: 'Total cross section of $t\bar{t}$ at 8 TeV'
```

Note that if you put text under latex `$` the spaces will be removed.

But (looking at the report) wouldn't it be better to have `r"CMS $\sigma_{\rm{tt}}^{\rm tot}$ 5 TeV"` ? I think it looks better.

**Review by scarlehoff — COMMENTED** [review]
Thanks for this! I've done a first test, I find some differences in the data. When this difference is expected please write something about it in the PR (in order to know whether it is a bug or a feature).

For Atlas only the 7 TeV one agrees for everything. I find some differences in the t0 chi2 which is pointing to a difference in the treatment or MULT and ADD?
```
 > Testing old: ATLASTTBARTOT7TEV vs new: ATLAS_TTBAR_7TEV_TOT_X-SEC
The t0 chi2 is different: 2194.2430166777112 vs 1102.1581162540726

 > Testing old: ATLASTTBARTOT8TEV vs new: ATLAS_TTBAR_8TEV_TOT_X-SEC
Data differences under 1e-2, continuing comparison...
	 242.4  242.9
# Differences in the computation of chi2 391.76 vs 548.71
    The covmats are different
    even the diagonal
    
 > Testing old: ATLASTTBARTOT13TEV vs new: ATLAS_TTBAR_13TEV_TOT_X-SEC
Relative differences above 1e-2                 
	 818.0   830 
```

For CMS instead I find perfect agreement for everything but the 13 TeV one, where there's a big difference in the results:
```
792.0   746 
```

So for now I've done the following report only for CMS: https://vp.nnpdf.science/WEzTkruoTt6kZNpn8igKdA==

**Comment by t7phy** [comment]
> Thanks for this! I've done a first test, I find some differences in the data. When this difference is expected please write something about it in the PR (in order to know whether it is a bug or a feature).
> 
> For Atlas only the 7 TeV one agrees for everything. I find some differences in the t0 chi2 which is pointing to a difference in the treatment or MULT and ADD?
> 
> ```
>  > Testing old: ATLASTTBARTOT7TEV vs new: ATLAS_TTBAR_7TEV_TOT_X-SEC
> The t0 chi2 is different: 2194.2430166777112 vs 1102.1581162540726
> 
>  > Testing old: ATLASTTBARTOT8TEV vs new: ATLAS_TTBAR_8TEV_TOT_X-SEC
> Data differences under 1e-2, continuing comparison...
> 	 242.4  242.9
> # Differences in the computation of chi2 391.76 vs 548.71
>     The covmats are different
>     even the diagonal
>     
>  > Testing old: ATLASTTBARTOT13TEV vs new: ATLAS_TTBAR_13TEV_TOT_X-SEC
> Relative differences above 1e-2                 
> 	 818.0   830 
> ```
> 
> For CMS instead I find perfect agreement for everything but the 13 TeV one, where there's a big difference in the results:
> 
> ```
> 792.0   746 
> ```
> 
> So for now I've done the following report only for CMS: https://vp.nnpdf.science/WEzTkruoTt6kZNpn8igKdA==

Actually I am pretty confident about the data and uncer. here because there is no data implementation here, everything is taken just from arxiv papers directly and I jut rechecked the values, it does appear to be consistent. Why does the old implementation differ? That I am not sure tbh.

File: buildmaster/CMS_TTBAR_8TEV_TOT/metadata.yaml (hunk 1 above)
Comment (t7phy) [reply buildmaster/CMS_TTBAR_8TEV_TOT/metadata.yaml:36]: Yes, I agree with your suggestion, I will change this for all the datasets. Thanks!

**Comment by scarlehoff** [comment]
There's a comment about a fix in the old implementation https://github.com/NNPDF/nnpdf/blob/master/nnpdfcpp/data/commondata/DATA_CMSTTBARTOT13TEV.dat

Maybe @enocera remembers

**Comment by felixhekhorn** [comment]
A (slightly) unrelated comment (but since I saw this in my newsfeed I decided to post it nevertheless - feel free to discard it as off-topic): I assume this is about fully inclusive ttbar xs (and the jets are in this PR by mistake), right?

I'm currently working on [MaunaKea](https://github.com/felixhekhorn/MaunaKea) a.k.a. "Hawaiian top++" patched on top of [`top++`](https://www.precision.hep.phy.cam.ac.uk/top-plus-plus/) by Czakon and Mitov for my work here in Jyväskylä - do we want to use this eventually on the theory side? The theory is currently provided by [mg5](https://github.com/NNPDF/pinecards/tree/master/ATLAS_TTB_7TEV_TOT) which means it is limited to NLO, but I could go full NNLO. The impact on the fit will be 0 since we talking about the grand total of 7 datapoints (associated to this PR).

**Comment by scarlehoff** [comment]
Yes, I think that will be great! I mean, just for consistency sake it is worth it.

**Comment by t7phy** [comment]
> A (slightly) unrelated comment (but since I saw this in my newsfeed I decided to post it nevertheless - feel free to discard it as off-topic): I assume this is about fully inclusive ttbar xs (and the jets are in this PR by mistake), right?
> 
The jets appear in the commits because the branch that is collecting the datasets already has the jets merged as they were considered finalized and done

> I'm currently working on [MaunaKea](https://github.com/felixhekhorn/MaunaKea) a.k.a. "Hawaiian top++" patched on top of [`top++`](https://www.precision.hep.phy.cam.ac.uk/top-plus-plus/) by Czakon and Mitov for my work here in Jyväskylä - do we want to use this eventually on the theory side? The theory is currently provided by [mg5](https://github.com/NNPDF/pinecards/tree/master/ATLAS_TTB_7TEV_TOT) which means it is limited to NLO, but I could go full NNLO. The impact on the fit will be 0 since we talking about the grand total of 7 datapoints (associated to this PR).

are you referring to the grids generation or ...? we are using Matrix for NNLO grids here, or maybe I completely misunderstood your point xD

**Comment by scarlehoff** [comment]
To be corrected in the list of legacy dataset names:

ATLAS_TTBAR_13TEV_TOT_X-SEC should be compared against ATLAS_TTBARTOT_13TEV_FULLLUMI

When doing this change, everything is ok.

The difference for the CMSTTBARTOT13TEV vs CMS_TTBAR_13TEV_TOT_X-SEC remains though and should be understood. The difference is  792.0   746 
We need to understand whether maybe CMS_TTBAR_13TEV_TOT_X-SEC and CMSTTBARTOT13TEV actually correspond to different datasets.

**Comment by t7phy** [comment]
@scarlehoff 
> The difference for the CMSTTBARTOT13TEV vs CMS_TTBAR_13TEV_TOT_X-SEC remains though and should be understood. The difference is 792.0 746 We need to understand whether maybe CMS_TTBAR_13TEV_TOT_X-SEC and CMSTTBARTOT13TEV actually correspond to different datasets.

So nnpdf uses https://arxiv.org/abs/1510.05302, which is exactly what I have used too:
the values on arxiv are:
v1: 769
v2 and v3: 746
I am unable to find 792 anywhere. Is 'CMSTTBARTOT13TEV' used in the 4.0 runcard?

**Comment by scarlehoff** [comment]
Yes
https://github.com/NNPDF/nnpdf/blob/72008b25884f65848fe9741b2842e2bc927638fc/n3fit/runcards/reproduce_nnpdf40/NNPDF40_nnlo_as_01180_1000.yml#L74

https://github.com/NNPDF/nnpdf/blob/master/nnpdfcpp/data/commondata/DATA_CMSTTBARTOT13TEV.dat

**Pull request closed without merging**